src/services/GoogleService.tsx

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const OAUTH2_REVOKE_API = "https://accounts.google.com/o/oauth2/revoke";
```

Comment:
Move this link to env file, so we can easily spot all the api-s we are integrating with.